codephile/lib/screens/profile/profile_skeleton.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'package:shimmer/shimmer.dart';
+
+Widget ProfileSkeleton(BuildContext context){
```

Comment:
Method names must be in `lowerCamelCase`.
For more details checkout :- https://dart.dev/guides/language/effective-dart/style